localstack/services/apigateway/helpers.py

```diff
@@ -141,6 +141,92 @@ def resolve_references(self) -> dict:
         return self._resolve_references(self.document)
 
 
+class IntegrationParameters(TypedDict):
+    path: Dict[str, str]
+    querystring: Dict[str, str]
+    headers: Dict[str, str]
+
+
+class RequestParametersResolver:
+    """
+    Integration request data mapping expressions
+    https://docs.aws.amazon.com/apigateway/latest/developerguide/request-response-data-mappings.html
+    """
+
+    def resolve(self, context: ApiInvocationContext) -> IntegrationParameters:
+        """
+        Resolve integration request parameters from the integration request parameters
```

Comment:
nit: this docstring seems a bit redundant, maybe we could concretize it a bit more (no need to change now, could be done in one of the next iterations.. 👍 )

Reply:
Agreed - let me fix that is something meaningful from AWS